doc/source/io.rst

```diff
@@ -20,9 +20,10 @@ Any arguments passed to ``read_file()`` after the file name will be passed direc
 Among other things, one can explicitly set the driver (shapefile, GeoJSON) with the ``driver`` keyword, or pick a single layer from a multi-layered file with the ``layer`` keyword.
 
 Where supported in ``fiona`` *geopandas* can also load resources directly from
-a web URL, for example for GeoJSON files from `geojson.xyz <http://geojson.xyz/>`.
+a web URL, for example for GeoJSON files from `geojson.xyz <http://geojson.xyz/>`_::
 
-    gpd.read_file("https://d2ad6b4ur7yvpq.cloudfront.net/naturalearth-3.3.0/ne_110m_land.geojson")
+    url = "http://geopandas.orgd2ad6b4ur7yvpq.cloudfront.net/naturalearth-3.3.0/ne_110m_land.geojson"
+    gpd.read_file(url)
```

Comment:
The url you changed it to isn't valid; it also isn't quite as clear anymore with the preceding sentence about reading a GeoJSON file from geojson.xyz (which is the case for the original url, though given that geojson.xyz doesn't appear in the url anywhere, it is not totally transparent then either).

Reply:
Whoops, I had started thinking about using a "geopandas.org" URL, then realized it would probably not be a nice URL either and concentrated on fixing the syntax errors.

I amended the commit and all should now work again!

Reply:
Did you push the change? I am still seeing the url with geopandas in the diff online?

Reply:
Just did, Monday morning ... (thanks for the heads-up!)